### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by DarkxDev**
- Set up a new React application using CRA.
- Added main reusable components.
- Connected with React Router

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @DarkxDev , 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have an updated README file.

- [ ] Please make sure all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.
- [ ] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.
- [ ] Please remove the Microverse logo from your README file


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/pages/Categories.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const Categories = () => (
+  <div>
+    <h2>Check status</h2>
+  </div>
+);
```
Comment (emmiiorji) [inline src/pages/Categories.js:5]: In order to allow for easy navigation to the books page, kindly include the nav component in the categories page.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @DarkxDev, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Linter checks are passing
- Proper use of gitflow
- Well documented


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Categories.js (hunk 1 above)
Comment (DarkxDev) [reply src/pages/Categories.js:5]: Hi @emmiiorji, the Navbar component is being rendered from the main App.js, before the Routes, and outside any other component so it doesn't get duplicated. This way, you can easily navigate the website to either the books or conference pages, by keeping the Navbar visible without re-rendering the component.

Applying this suggestion would go against the DRY principle.

Comment (emmiiorji) [reply src/pages/Categories.js:5]: Hi @DarkxDev

Sorry, that was an oversight, you're right on this.

Thanks for pointing that out. Kindly request for review again on your dashboard and the next will justice.

**Review by Nemwel-Boniface — COMMENTED** [review]
Hello @DarkxDev  👋,

I am Nemwel your reviewer.


## _This is an additional review as a result of the second opinion request._

Because the requested changes were asked for functionality that was already implemented, Kudos for adhering to the DRY principle. your project can be approved :rocket: .

**Note:**
- [ ] However, I highly suggest that you should make sure that the Categories page has a button that says "Check status".

- [ ]  Also, kindly ensure that you update your MIT file as it is currently saying "Math magicians" and this is a different project.

As this was the only reason that your project was not approved please submit it again in your dashboard and the next reviewer will approve it immediately.

Keep rocking 🚀!

**Review by TadesseDev — APPROVED** [review]
### Hi @DarkxDev 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**